src/model/src/FreeFloatingState.cpp

```diff
@@ -100,7 +100,7 @@ const JointDOFsDoubleArray& FreeFloatingVel::jointVel() const
 void FreeFloatingVel::resize(const Model& model)
 {
     this->m_baseVel.zero();
-    this->m_jointVel.resize(model.getNrOfDOFs());
+    this->m_jointVel.resize(model.getNrOfPosCoords());
```

Comment:
This is wrong. The `m_jointVel` and `m_jointAcc` vectors should have a size `model.getNrOfDOFs()`. For example, for a model with just a RevoluteSO2 joint, the size of `m_jointVel` should be 1, not 2.